tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/publicAsset/PublicAssetSetRequest.scala

```diff
@@ -115,6 +115,7 @@ case class PublicAssetCreationSuccess(publicAssetCreationId: PublicAssetCreation
 case class PublicAssetCreationFailure(publicAssetCreationId: PublicAssetCreationId, exception: Throwable) extends PublicAssetCreationResult {
   def asPublicAssetSetError: SetError = exception match {
     case e: PublicAssetCreationParseException => e.setError
+    case e: PublicAssetQuotaLimitExceededException => SetError.overQuota(SetError.SetErrorDescription(e.getMessage))
```

Comment:
BTW where are logs in this class ?

 - > parse exceptions are lost
 - > quota exceeded are insteresting info events IMO...

Reply:
> > parse exceptions are lost

it is `PublicAssetCreationParseException` and `PublicAssetException` , and already handled